src/Sylius/Component/Payment/Tests/Encryption/PaymentRequestEncrypterTest.php

```diff
@@ -0,0 +1,397 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\Payment\Encryption;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use stdClass;
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Payment\Encryption\EncrypterInterface;
+use Sylius\Component\Payment\Encryption\EntityEncrypterInterface;
+use Sylius\Component\Payment\Encryption\PaymentRequestEncrypter;
+use Sylius\Component\Payment\Model\PaymentRequestInterface;
+
+final class PaymentRequestEncrypterTest extends TestCase
+{
+    private MockObject $encrypterMock;
+
+    private PaymentRequestEncrypter $paymentRequestEncrypter;
+
+    protected function setUp(): void
+    {
+        $this->encrypterMock = $this->createMock(EncrypterInterface::class);
+        $this->paymentRequestEncrypter = new PaymentRequestEncrypter($this->encrypterMock);
+    }
+
+    public function testAnEntityEncrypter(): void
+    {
+        $this->assertInstanceOf(EntityEncrypterInterface::class, $this->paymentRequestEncrypter);
+    }
+
+    public function testDoesNothingWhenEncryptingPaymentRequestWithNoPayloadAndEmptyResponseData(): void
+    {
+        $paymentRequestMock = $this->createMock(PaymentRequestInterface::class);
```

Comment:
As it is used in multiple tests, it can be created in `setUp` method, the same should be checked in other tests not only in this file